src/pages/Checkout.page.js

```diff
@@ -0,0 +1,40 @@
+class CheckoutPage {
+    constructor(page) {
+        this.page = page;
+    }
+
+    async fillCheckoutInformation(firstName, lastName, postalCode) {
+        await this.page.fill('#first-name', firstName);
+        await this.page.fill('#last-name', lastName);
+        await this.page.fill('#postal-code', postalCode);
+    }
+
+    async continue() {
+        await this.page.click('#continue');
+        await this.page.waitForLoadState('networkidle'); 
+    }
+
+    async getCheckoutItems() {
+        const items = await this.page.$$('.checkout_item');
+        return Promise.all(items.map(async item => ({
+            name: await item.$eval('.inventory_item_name', el => el.innerText),
```

Comment:
you can use:
```suggestion
            name: await item.locator('.inventory_item_name').innerText(),
```
that would be much easier :)